src/components/SideNavBar/style.scss

```diff
@@ -30,25 +30,21 @@
   position: relative;
   border-bottom: 1px solid rgba(0,0,0,0.3);
   transition: all 0.3s;
-  &:hover {
-    background: darken($cPurple100, 5%);
-  }
-  &:active {
-    background: darken($cPurple100, 10%);
+
+  &:hover, &:active {
```

Comment:
Would be nice to have `&:focus` as well, as it's usually paired with `&:hover` for accessibility reasons